src/libultra/io/motor.c

```diff
@@ -1,12 +1,18 @@
+#include "ultra64/motor.h"
+
+#include "io/controller.h"
+
```

Comment:
Why the blank lines?

Reply:
ultra64.h and global.h were already here.
I think ultra64.h could be removed, but global.h is needed, because it needs the functions prototypes from another files. I'll try changing it to functions.h